include/seqan3/io/sam_file/record.hpp

```diff
@@ -152,28 +152,22 @@ class sam_record : public record<field_types, field_ids>
         return get_impl(field_constant<seqan3::field::offset>{}, static_cast<tuple_base_t const &>(*this));
     }
 
-    /*!\brief The (pairwise) alignment stored in an object that models seqan3::detail::pairwise_alignment.
-     * \returns seqan3::sam_file_input::alignment_type per default
+    /*!\brief [DEPRECATED] The (pairwise) alignment stored in an object that models seqan3::detail::pairwise_alignment.
+     *
+     * This member is deprecated. Please access `cigar()` and then use `seqan3::alignment_from_cigar` to retrieve the
+     * alignment.
      */
-    decltype(auto) alignment() &&
-    {
-        return get_impl(field_constant<seqan3::field::alignment>{}, static_cast<tuple_base_t &&>(*this));
-    }
+    SEQAN3_DEPRECATED_340 decltype(auto) alignment() &&
+    {}
```

Comment:
Is there a way to dispatch to `alignment_from_cigar`, i.e. do we get access to the reference sequence from here somehow?

Reply:
no, since
1) It would require some dirty hacking to give extra data to the tuple (record) 
2) there is no guarantee that a reference sequence is provided.